pypeman/msgstore.py

```diff
@@ -140,6 +152,9 @@ async def search(self, **kwargs):
     async def total(self):
         return 0
 
+    async def delete(self, id):
+        return None
```

Comment:
doc strings:

ex.

""" nothing to delete therefore empty """

Reply:
I think the class docsting is sufficient, it's a "fake" class that do nothing